test/dlc_tests/eks/tensorflow/inference/test_eks_tensorflow_inference.py

```diff
@@ -9,8 +9,8 @@
 import test.test_utils as test_utils
 
 
-@pytest.mark.model("half_plus_two")
-def test_eks_tensorflow_half_plus_two_inference(tensorflow_inference):
+@pytest.mark.model("albert")
```

Comment:
any reason we are removing half plus two completely?

Reply:
> any reason we are removing half plus two completely?

we want to include more models for testing, and we already have ecs testing on half_plus_two